### ISSUE TITLE
Cannot pay using OVO with new Payments API

### ISSUE THREAD

**Issue by kuriizz** [xendit/xendit-node#207]
Hi Team,

Xendit SDK version used: 3.2.0

- Issue
I tried to create payment request using Payment Request function in the SDK with this body:
```
{
    currency: 'IDR',
    amount: amountToPay,
    referenceId: refId,
    paymentMethod: {
        reusability: 'ONE_TIME_USE',
        type: 'EWALLET',
        ewallet: {
            channelCode: 'OVO',
            channelProperties: {
                mobilenumber: '+6281234930058',
                successReturnUrl: successRedirectUrl,
                failureReturnUrl: failureRedirectUrl,
            },
        },
    },
}
```

and I use function `PaymentRequest.createPaymentRequest` to send the payment request. But I got this error `Invalid channel_properties for ID_OVO. Expecting mobile_number to be provided`.

But it works for `SHOPEEPAY` using the same body except I put `channelCode` as `SHOPEEPAY`.

- Workaround
I've tried using the API manually to this API `https://api.xendit.co/payment_requests`, with this body:
```
{
    "amount": 1500,
    "currency": "IDR",
    "country": "ID",
    "payment_method": {
        "type": "EWALLET",
        "ewallet": {
            "channel_code": "OVO",
            "channel_properties": {
                "mobile_number": "+6281239004977",
                "success_return_url": "https://redirect.me/goodstuff"
            }
        },
        "reusability": "ONE_TIME_USE"
    }
}
```
And it worked!

Is there something I missed in the flow of the new Payments API?
or there is a bug in the SDK?

Please help.
Thanks.

**Comment by willypt** [comment]
Hi @kuriizz,  based on the given documentation https://github.com/xendit/xendit-node/blob/master/docs/payment_method/models/EWalletChannelProperties.md, you may try adjusting 
```
            channelProperties: {
                mobilenumber: '+6281234930058',
                successReturnUrl: successRedirectUrl,
                failureReturnUrl: failureRedirectUrl,
            },
```

to 
```
            channelProperties: {
                mobileNumber: '+6281234930058',
                successReturnUrl: successRedirectUrl,
                failureReturnUrl: failureRedirectUrl,
            },
```

Let me know if that doesn't solve your issue

**Comment by kuriizz** [comment]
Hi @willypt, the thing is I want to create `PaymentRequest` not `PaymentMethod`, and the interface for the method parameter is from this documentation https://github.com/xendit/xendit-node/blob/master/docs/payment_request/models/EWalletChannelProperties.md.

But, I tried that yesterday anyway, even though I got an error from Typescript's typesafe checking. But it still doesn't work. I still got the same error as before.

**Comment by willypt** [comment]
whoops, sorry. https://github.com/xendit/xendit-node/blob/master/docs/payment_method/models/EWalletChannelProperties.md it also uses the same mobileNumber format.

**Comment by willypt** [comment]
Understood. Let me check and revert to you in this matter. 👍🏻

**Comment by willypt** [comment]
Hi @kuriizz , we identified a wrongly defined field for the mobilenumber. Our team will be prioritizing the fix and will release the fix asap 🙏🏻
cc: @adityarx

**Comment by kuriizz** [comment]
Ok, thank you so much 🙏🏻

**Comment by adityarx** [comment]
@kuriizz apologies for the delay, this issue has now been patched in v3.3. Please try, thank you!

**Comment by kuriizz** [comment]
Hi @adityarx, it works now! Thanks for the efforts you and the team have put into this.
